### DISCUSSION TITLE
hello!!

### DISCUSSION THREAD

**Discussion by CaiusCosades** [searx/searx#2869] [General]
how to add searx to firefox? on android
thank you!!!

**Comment by allendema** [comment]
Assumming that you are using the latest version of FF (89.1.1), this can be done by going to
Settings >> Search >> Add search Engine

and then depending on the instance you add one of the following options.

``` https://searx.example.org/?q=%s ```
``` https://searx.example.org/searx/?q=%s ```

If you want to search directly with a specific engine you can add it to the string, like so:

``` https://searx.example.org/?q=!sp !you !%s ```
